### ISSUE TITLE
Bar plot does not respect axes.prop_cycle in matplotlib 1.5

### ISSUE THREAD

**Issue by MaxNoe** [pydata/pandas#11735]
Using pandas 0.17.1 with matplotlib 1.5:

```{python}
import pandas as pd
import matplotlib.pyplot as plt
plt.style.use('ggplot')

data = pd.DataFrame({'1': [1, 2, 3], '2': [3, 5, 1]})

data.plot(kind='bar')
plt.savefig('test.png', dpi=300)
```
Results in 

![test](https://cloud.githubusercontent.com/assets/5488440/11506494/0328c792-9850-11e5-97ec-044a8ec2183f.png)

Which uses the colors of matplotlibs default style, not of `ggplot`.

Using matplotlib 1.4.3 the correct colors are used:
![result](https://cloud.githubusercontent.com/assets/5488440/11506670/e40350c0-9850-11e5-9495-83185673a54e.png)

**Comment by rcarneva** [comment]
Duplicate of #11614 and fixed by #11865

**Comment by TomAugspurger** [comment]
Thanks.
